src/api/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import './CommentList.css';
+
+const CommentList = ({ comments }) => (
+  <>
+    {comments.map(comment => (
+      <ul className="comments">
+        <div className="comments-name">
+          {comment.name}
+        </div>
+        <div className="comments-body">
+          {comment.body}
+        </div>
+        <div className="comments-email">
+          {comment.email}
+        </div>
+      </ul>
+    ))}
+  </>
```

Comment:
Make `ul` a parent component for all comments. Also, it doesn't make sense to use `ul` without `li`.